### ISSUE TITLE
Localstack not working on windows with python 3.9 

### ISSUE THREAD

**Issue by onlyabhi52** [localstack/localstack#3153]
# Type of request: This is a ...

[ x] bug report

[ ] feature request

# Detailed description

... This happens when we install localstack through python command and start it using localstack start . This is happening on multiple machines and the same error message comes irrespective of python versions.

## Expected behavior

... application should start as it happens using docker.

## Actual behavior

```
'uname' is not recognized as an internal or external command,
operable program or batch file.
Traceback (most recent call last):
  File "C:\Python39\Scripts\localstack", line 18, in <module>
    cli.main()
  File "C:\Python39\lib\site-packages\localstack\utils\cli.py", line 130, in main
    bootstrap.load_plugins(scope=bootstrap.PLUGIN_SCOPE_COMMANDS)
  File "C:\Python39\lib\site-packages\localstack\utils\analytics\profiler.py", line 153, in wrapped
    from localstack.utils.common import now_utc
  File "C:\Python39\lib\site-packages\localstack\utils\common.py", line 4, in <module>
    import grp
ModuleNotFoundError: No module named 'grp'
```

# Steps to reproduce
 localstack start command through powershell.
## Command used to start LocalStack

``` localstack start ```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

``` 
below is the list of installed pip packages : 
Package               Version
--------------------- ---------
boto3                 1.16.2
botocore              1.19.2
certifi               2020.6.20
chardet               3.0.4
collective.recipe.grp 1.1.0
dnslib                0.9.14
dnspython             1.16.0
docopt                0.6.2
idna                  2.10
jmespath              0.10.0
localstack            0.12.0
localstack-client     1.5
localstack-ext        0.11.43.6
pip                   20.2.4
pyaes                 1.6.1
python-dateutil       2.8.1
requests              2.24.0
s3transfer            0.3.3
setuptools            49.2.1
six                   1.15.0
urllib3               1.25.11
zc.buildout           2.13.3

**Comment by AlexisWilke** [comment]
Have you tried to use the Docker instead? As far as I know that's the way to make it work these days.

**Comment by onlyabhi52** [comment]
@AlexisWilke  it works fine with a docker compose file but with python it fails directly.

**Comment by AlexisWilke** [comment]
@onlyabhi52 You probably need very specific versions of each dependency which they have in their Docker. So you probably would have to check that to get the list that they use and it's probably not easy to get that straighten up (I don't think you have an easy way like with Node.js to select the version of each dependency?)

**Comment by whummer** [comment]
Thanks for reporting @onlyabhi52 @AlexisWilke . The import of the `grp` module has been removed in 1b3ce97d966c56349e8477e722142512e990bab0. We have just released a new version `0.12.1` - can you please try upgrading your LocalStack command line:
```
pip install --upgrade 'localstack>=0.12.1'
```
Hope that helps - please let us know if the problem persists. Thanks

**Comment by onlyabhi52** [comment]
this is working fine now @whummer
